src/event/create-event.dto.ts

```diff
@@ -0,0 +1,31 @@
+import { Transform } from 'class-transformer';
+import { IsInt, IsISO8601, IsNotEmpty, IsString, IsUrl } from 'class-validator';
+
+export class CreateEventDto {
+  @IsString()
+  @IsNotEmpty()
+  name: string;
+
+  @IsString()
+  @IsNotEmpty()
+  description: string;
+
+  @IsUrl()
+  @IsNotEmpty()
+  registration_link: string;
+
+  @IsISO8601()
+  @IsNotEmpty()
+  start_time: string;
+
+  @IsISO8601()
+  @IsNotEmpty()
+  end_time: string;
+
```

Comment:
Pa add din pala ako validation to check that `start_time` is earlier than `end_time`